src/main/webapp/app/exercises/shared/plagiarism/plagiarism-split-view/plagiarism-split-view.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="plagiarism-split-view">
+    <div class="plagiarism-split-pane" jhiPane>
+        <div class="plagiarism-split-pane-header">{{ comparison?.element1.studentLogin }}</div>
+        <jhi-modeling-editor *ngIf="submission1" [umlModel]="submission1?.model" [diagramType]="'ClassDiagram'" [readOnly]="true"></jhi-modeling-editor>
```

Comment:
ClassDiagram is hard-coded here. Do you have access to the exercise? You could use `exercise.diagramType` then.

Reply:
Great point, forgot about that. Added an input binding to `PlagiarismSplitViewComponent`, check the latest commit 🙂